package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "gpii-grunt-mdjson-lint",
+  "description": "A plugin to lint JSON and JSON5 blocks within markdown documents.",
+  "version": "1.0.0",
+  "homepage": "",
+  "author": "Tony Atkins <tony@raisingthefloor.org>",
+  "repository": {
+    "type": "git",
+    "url": "git://gihub.com/the-t-in-rtf/grunt-mdjson-linter.git"
```

Comment:
Please update to core repo URL (also note that repo name itself is incorrect)

Reply:
It's horribly incorrect, I spelled "gihub" wrong as well.  Anyway, fixed.